domains/creed.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
I cannot reach your website, 404 page

(if you want to use github pages through A record, it's 4 IPs
- `185.199.111.153`
- `185.199.110.153`
- `185.199.109.153`
- `185.199.108.153`
)